cico_pr_build.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
```

Comment:
This will fail the build if any command fails.
```suggestion
set -e

```